getYear.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"errors"
+	"github.com/rwcarlsen/goexif/exif"
+	"log"
+	"os"
+)
+
+/**
+ * This function grabs the year from the file camera data
+ */
+func getYearFromExif(filePath string) (int, error) {
+	// Open the file, which is required for extracting
+	// exif data (which is camera specific file attributes)
+	file, err := os.Open(filePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	exifData, err := exif.Decode(file)
+	if err != nil {
+		return 0, errors.New("No exif data avaliable")
```

Comment:
```suggestion
		return 0, errors.New("No exif data available")
```

Reply:
I suppose using [error wrapping](https://go.dev/blog/go1.13-errors) might be good here in case the error is truly unexpected (e.g. your disk has failed).